ext/project_data.yaml

```diff
@@ -9,9 +9,10 @@ version_file: 'lib/puppet/version.rb'
 # files and gem_files are space separated lists
 files: '[A-Z]* install.rb bin lib conf man examples ext tasks spec locales'
 # The gem specification bits only work on Puppet >= 3.0rc, NOT 2.7.x and earlier
-gem_files: '[A-Z]* install.rb bin lib conf man examples ext tasks spec locales'
+gem_files: '[A-Z]* install.rb bin lib libbolt conf man examples ext tasks spec locales'
 gem_test_files: 'spec/**/*'
 gem_executables: 'puppet'
```

Comment:
This causes the gem to have executable conflicts with puppet when installed as a gem alongside puppet. I presume we should namespace or remove the executable.